tools/bazel.rc

```diff
@@ -1,6 +1,8 @@
 build:cuda --crosstool_top=@local_config_cuda//crosstool:toolchain
 build:cuda --define=using_cuda=true --define=using_cuda_nvcc=true
 
+build:mkl --define=using_mkl=true
+
```

Comment:
Thanks! Also noticed from: https://github.com/tensorflow/tensorflow/blob/master/tools/bazel.rc

# This config option is used to enable MKL-DNN open source library only,
# without depending on MKL binary version.
build:mkl_open_source_only --define=using_mkl_dnn_only=true

Seems like a good option here as well?